src/components/map/Measurement.js

```diff
@@ -0,0 +1,146 @@
+import { Group, Vector3 } from 'three'
+import MultiLine from '@/components/map/layers/MultiLine'
+import Highlight from '@/components/map/layers/Highlight'
+
+const CENTER_X = 4.549
+const CENTER_Y = -44.632
+const RENDERED_RADIUS = 2125 / (2 * Math.PI)
+const INV_RENDERED_RADIUS = 1 / RENDERED_RADIUS
+const realRadius = 5663
```

Comment:
Just wondering how did you get this `realRadius`👻

Reply:
This value comes from the circumference in miles we learned here: https://wob.coppermind.net/events/332/#e10231

Reply:
Got it! Thank you~~😁